README.md

```diff
@@ -50,6 +64,8 @@ file syntax is explained in the next section.
 
 The current MIDIMonster version can be queried by passing *-v* as command-line argument.
 
+___
```

Comment:
Visually "fences in" the following section heading. IMO that hurts readability more than it helps